src/GraphQL/Validation/Errors/NoUndefinedVariablesError.cs

```diff
@@ -19,7 +19,7 @@ public NoUndefinedVariablesError(ValidationContext context, Operation node, Vari
 
         internal static string UndefinedVarMessage(string varName, string opName)
             => !string.IsNullOrWhiteSpace(opName)
-                ? $"Variable \"${varName}\" is not defined by operation \"{opName}\"."
-                : $"Variable \"${varName}\" is not defined.";
+                ? $"Variable '${varName}' is not defined by operation '{opName}'."
+                : $"Variable '${varName}' is not defined.";
```

Comment:
Simplify?

Reply:
Maybe not here...

Reply:
It is better to save readability, I inverted the condition.